packages/stores/types.ts

```diff
@@ -66,7 +66,7 @@ type TRoutes =
     | '/reports/positions'
     | '/reports/profit'
     | '/reports'
-    | '/'
+    | '/dtrader'
```

Comment:
@ahmadtaimoor-deriv I guess you don't need to delete `/` route here, just add `/dtrader` and keep `/` route too, because we still have `routes.root` in this file (packages/shared/src/utils/routes/routes.ts)